composer.json

```diff
@@ -20,7 +20,7 @@
     ],
 
     "require": {
-        "php":                       ">=5.3.1",
+        "php":                       ">=5.4",
```

Comment:
EOL or not, but we still support it.

Reply:
Do you think it's worth it? Is there any stats about 5.3 usage?
Also we are locking us down with old version of PHP and because of this it's blocking us with:
- Symfony 4 package
- Facebook WebDriver migration (my another PR) to become more w3c compatible

Reply:
Supporting PHP 5.3 doesn't cost us anything currently.

Reply:
Please restore that code, that used different Ubuntu image to test with PHP 5.3.

Reply:
Will do but keep in mind that Docker will not work on Precise

Reply:
You said, that you'll restore PHP 5.3 support, but you haven't and marked whole conversation as resolved.

Reply:
It's quite hard to sync multiple changes.
I do not want to restore 5.3 because it's not worth it.
https://github.com/minkphp/driver-testsuite/pull/37